### PR TITLE
Release draft

### PR DESCRIPTION
**Pull request by sumanthkb44**
Updated Readme.md file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -81,9 +60,35 @@ docker run --rm -it /path/to/InputDirectory:/mnt/Input -v /path/to/OutputDirecto
...
+ 2. **SW360 Package Creator** - This executable expects the `CycloneDX BOM` as the input, creates the missing components/releases in SW360 and links all the components to the respective project in SW360 portal and triggers the fossology upload.
+ 
+ ```text
+  SW360PackageCreator.exe --settingsfilepath /<Config_Path>/appSetting.json
+```
+ 3. **Artifactory Uploader** - This executable takes `CycloneDX BOM` which is updated by the ` SW360PackageCreator.dll` as input and uploads the components that are already cleared (clearing state - "Report approved") to the SIPARTY release repo in Jfrog Artifactory.
+ ```text
+   ArtifactoryUploader.exe --settingsfilepath /<Config_Path>/appSetting.json
+  ```
+
```
Comment (karthika-g) [inline README.md:85]: I think it is better to add argument list here

File: README.md [hunk 2]
```diff
@@ -16,32 +16,32 @@ manual effort of creating SW360 and FOSSology workflows.
...
- # Usage
  
-#### Package Installation 
-
- In order to consume the CA docker image directly, follow the steps below:
-
- 1. Clone the repo to your local machine 
- 2. Build the source code
- 3. Create an image using the command below
+## Package Installation
```
Comment (karthika-g) [inline README.md:20]: Ensure all the main heading are having the same font size

File: README.md [hunk 3]
```diff
@@ -81,9 +60,35 @@ docker run --rm -it /path/to/InputDirectory:/mnt/Input -v /path/to/OutputDirecto
  ```text
   docker run --rm -it /path/to/InputDirectory:/mnt/Input -v /path/to/OutputDirectory:/mnt/Output -v /path/to/LogDirectory:/var/log -v /path/to/configDirectory:/etc/CATool sw30clearingautomationtool dotnet ArtifactoryUploader.dll --settingsfilepath /etc/CATool/appSetting.json
   ```
+</details>
+
+<details>
+<summary>Run as binaries</summary>
+ 
+ Extact the downloaded nupkg package , move to tools folder and execute the below commands in order,
```
[the remark this answers is not in the document]
Comment (karthika-g) [reply README.md:68]: correct extract spelling also

File: README.md [hunk 4]
```diff
@@ -16,32 +16,32 @@ manual effort of creating SW360 and FOSSology workflows.
...
- ### Execution via terminal
-
-The Clearing Automation Tool has 3 dll 's.
-
-Execute them in the following order to achieve the complete License clearing process.
+```bash
+docker pull ghcr.io/siemens/continuous-clearing:v3.0.21
+ ```
 
+#### Use dotnet pacakge
```
[the remark this answers is not in the document]
Comment (sumanthkb44) [reply README.md]: Checked

File: README.md [hunk 5]
```diff
@@ -16,32 +16,32 @@ manual effort of creating SW360 and FOSSology workflows.
...
+ ### Install from GitHub Release (Official)
+#### Use container image
 
- ### Execution via terminal
-
-The Clearing Automation Tool has 3 dll 's.
-
-Execute them in the following order to achieve the complete License clearing process.
+```bash
+docker pull ghcr.io/siemens/continuous-clearing:v3.0.21
```
[the remark this answers is not in the document]
Comment (sumanthkb44) [reply README.md]: Checked

**Review by karthika-g — APPROVED** [review]
Looks good now

**Pull request merged**